partner_firstname/tests/test_name.py

```diff
@@ -50,6 +53,19 @@ def test_whitespace_cleanup(self):
         # Need this to refresh the ``name`` field
         self.original.invalidate_recordset(["name"])
 
+    def test_multiple_name_creation(self):
+        """Create multiple partners at once, only with "name"."""
+        self.env["res.partner"].create(
+            [
+                {
+                    "name": "Test partner1",
+                },
+                {
+                    "name": "Test partner2",
+                },
+            ]
+        )
+
 
```

Comment:
Provide atleast one assertion, e.g. `assertRecordValues`

Reply:
Assertion added; there was none before because I was already happy that no exception is raised, as it happens before these changes.